src/writer.rs

```diff
@@ -176,6 +176,8 @@ mod write_vtk_impl {
             DataSet(DataSetError),
             NewLine,
 
+            InvalidColorScalarsType,
```

Comment:
It seems that this error is not used anywhere. Maybe it could be used in the `write_color_scalars_buf` functions below? Otherwise we should remove it I think.